examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
okay, so what do we do? We could also pass a bool param called `unmark_bads` which is True by default.